### DISCUSSION TITLE
Tailwind CSS Docs Workflow for Alfred on macOS

### DISCUSSION THREAD

**Discussion by techouse** [tailwindlabs/tailwindcss#3236] [Show and tell]
Hi,

So about 2 years ago when I first dipped into Tailwind I noticed that I was basically glued to the documentation website 99% of the time 😆 So inspired by some other tools out there I wrote a small Python app that plugs into Alfred and uses the same Algolia index that the TailwindCSS docs use. It sped up my development a bit since I'm quite used to the whole Alfred workflow logic.

Anyway, maybe it can help some of you out, so grab it here if you'd like :) https://github.com/techouse/alfred-tailwindcss-docs

Cheers 🚀

**Comment by Parisiana** [comment]
Thanks for the update and quick reply. I'll be sure to keep an eye on this thread. Looking for the same issue. Bumped into your thread. Thanks for creating it. Looking forward for solution.
	

[MyIndigoCard](https://www.myindigocard.us/)
